portal/components/connectWallets/wallets.tsx

```diff
@@ -58,8 +58,8 @@ const ConnectWalletButton = function ({
       {icon}
       <span className="text-base font-medium text-neutral-950">{text}</span>
       {rightIcon || (
-        <div className="ml-auto">
-          <Chevron.Right />
+        <div className="group-hover ml-auto">
+          <Chevron.Right className="group-hover:[&>path]:fill-neutral-950" />
```

Comment:
I think this is incorrect. Check out the [group-hover](https://v3.tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-parent-state) docs

```suggestion
        <div className="group ml-auto">
          <Chevron.Right className="group-hover:[&>path]:fill-neutral-950" />
```

The parent uses `group` and the children uses `group-hover`. They can also be named, like 


```suggestion
        <div className="group/container ml-auto">
          <Chevron.Right className="group-hover/container:[&>path]:fill-neutral-950" />
```

but apparently that's not mandatory

Reply:
Fixed here 3d67e7a7c9bf96bd276d8319e7c91fe8ccd79f89. Thanks!